apollo-router/src/executable.rs

```diff
@@ -62,6 +62,10 @@ pub(crate) struct Opt {
     )]
     config_path: Option<PathBuf>,
 
+    /// Enable development mode.
+    #[clap(env = "APOLLO_ROUTER_DEV_MODE", long = "dev", hide(true))]
+    dev: bool,
+
```

Comment:
I wonder if we should go with something just a bit shorter `APOLLO_ROUTER_DEV`.  Matches the flag better anyhow?

```suggestion
    #[clap(env = "APOLLO_ROUTER_DEV", long = "dev", hide(true))]
    dev: bool,

```

Reply:
cc @EverlastingBugstopper this suggestion has been applied